share/functions/__fish_cancel_commandline.fish

```diff
@@ -3,16 +3,10 @@ function __fish_cancel_commandline
     set -l cmd (commandline)
     if test -n "$cmd"
         commandline -C 1000000
-        # TODO: Switch from using tput and standout mode to `set_color` when themes have been
-        # augmented to include support for background colors or has support for standout/reverse
-        # mode.
-        #
-        # Set reverse fg/bg color mode, output ^C, restore normal mode, clear to EOL (to erase any
-        # autosuggestion).
+        echo -ns (set_color $fish_color_cancel) "^C" (set_color normal)
         if command -sq tput
-            echo -ns (set_color -r) "^C" (set_color normal) (tput el; or tput ce)
-        else
-            echo -n "^C"
+            # Clear to EOL (to erase any autosuggestion).
+            echo -n (tput el; or tput ce)
```

Comment:
Shouldn't this be `echo -n (tput el ^/dev/null; or tput ce)`? After all, if `el` isn't legal an error is going to be written to stderr.